packages/ui/src/components/chat.tsx

```diff
@@ -0,0 +1,149 @@
+import { ChangeEvent, KeyboardEvent, ReactNode, RefObject } from 'react'
+
+import { Button, Icon, Input, Text } from '@/components'
+import { cn } from '@/utils/cn'
+
+// Root Container
+const Root: React.FC<{ children: ReactNode }> = ({ children }) => {
```

Comment:
`React` isn't imported in this file, you can add `FC` to the existing imports.
```suggestion
const Root: FC<{ children: ReactNode }> = ({ children }) => {
```